src/mono/wasm/wasm.proj

```diff
@@ -240,8 +240,8 @@
     <PropertyGroup>
       <PInvokeTableFile>$(ArtifactsObjDir)wasm/pinvoke-table.h</PInvokeTableFile>
       <InterpToNativeTableFile>$(ArtifactsObjDir)wasm/wasm_m2n_invoke.g.h</InterpToNativeTableFile>
-      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Debug'">-g -Os -s -DDEBUG=1 -DENABLE_AOT_PROFILER=1</CMakeConfigurationEmccFlags>
-      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Release'">-Oz</CMakeConfigurationEmccFlags>
+      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Debug'">-g -Os -s -DDEBUG=1 -DENABLE_AOT_PROFILER=1 -DENABLE_BROWSER_PROFILER=1</CMakeConfigurationEmccFlags>
+      <CMakeConfigurationEmccFlags Condition="'$(Configuration)' == 'Release'">-Oz  -DENABLE_BROWSER_PROFILER=1</CMakeConfigurationEmccFlags>
```

Comment:
This will enable the profile by default in release mode, is this what we want ?

Reply:
Yes, if it's just bytes in the `dotnet.wasm` 
It would not make the interp slower unless `mono_profiler_init_browser` is called.
It would not make the AOT to emit icalls unless `<WasmProfilers>browser;</WasmProfilers>` is also set.

Did I get it right ?

Reply:
2512425 bytes vs  2512667 of `artifacts\bin\native\net7.0-Browser-Release-wasm\dotnet.wasm` 
just that `-DENABLE_BROWSER_PROFILER=1` difference for Release
That is 242 just bytes. 😲